glide.lock

```diff
@@ -1,5 +1,5 @@
-hash: 44c1b4016b4c802a043b2955cb24e2922ae9949f2ebcbbb0351529107d9d2f11
-updated: 2017-11-16T16:14:50.837044899-08:00
+hash: 2ff54d6684eff3a6f08b8735dade82b31597521ada7ef5a87002a8c5dd13acfd
```

Comment:
Let's keep glide updates as separate commits.

Reply:
This is needed to pick up the idl update.